src/dataDisplay/Number/index.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+
+// https://github.com/gnosis/safe/wiki/How-to-format-amounts
+const DEFAULT_SHOW_SIGN = false;
+
+type FormatOptions = {
+  currency?: string;
+  showSign?: boolean;
+};
+
+type IntlFormatNumberOptions = Intl.NumberFormatOptions & {
+  notation?: 'compact'; // M, B, T
+  signDisplay?: 'always' | 'auto'; // +/-
+};
+
+export const formatGnosisNumber = (
+  value: string,
+  { currency, showSign = DEFAULT_SHOW_SIGN }: FormatOptions = {}
+): string => {
+  const number = +value;
+
+  if (isNaN(number)) {
+    return value;
+  }
+
+  const formatNumber = (
+    number: number,
+    format: IntlFormatNumberOptions,
+    prefix = ''
+  ) => {
+    return (
+      (prefix && showSign ? `${prefix} ` : prefix) + // Add space if displaying +/- signs
+      new Intl.NumberFormat([], {
+        signDisplay: showSign ? 'always' : 'auto', // Positive symbol
```

Comment:
Not sure I follow the logic with the sign and prefix. Isn't it also adding the sign one line 32?

Reply:
Sorry, my comment on line 32 is not clear enough. The prefix is not a sign, but i.e. '>'